.gitignore

```diff
@@ -1,6 +1,8 @@
 HELP.md
-resources/
 target/
+application.properties
+liquebase.properties
```

Comment:
Typo, and why do you exclude both application and liquibase properties from versioning?